logger/common.go

```diff
@@ -495,9 +490,13 @@ func SetUIDAndGID(uid int, gid int) error {
 }
 
 // setUID sets UID of current goroutine.
```

Comment:
nit: Can you update this comment to be the same as line 514 (`setUID sets UID of current goroutine/process.`)

Reply:
Sure let me update the comment